linuxNew/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -1,4 +1,4 @@
-%global upstream_version 8.0.302+8
+%global upstream_version 8.0.302-b08
```

Comment:
This should be `8u302-b08` there is no 8.0.302-b08 upstream version ;-) The source URL would not point to an existing URL. Check with `rpmspec -P temurin-8-jdk.spec | grep Source0`. The result should point to the real binary.